### PR TITLE
Fixing static check due to faulty entry in openapi spec

### PR DESCRIPTION
**Pull request by amoghrajesh**
Fixing CI failure on https://github.com/apache/airflow/actions/runs/11945173632/job/33297606334. 

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by amoghrajesh** [comment]
Looks like the original one is the correct one

File: airflow/ui/openapi-gen/requests/schemas.gen.ts [hunk 1]
```diff
@@ -4290,7 +4290,7 @@ export const $TimeDelta = {
   properties: {
     __type: {
       type: "string",
-      title: "  Type",
+      title: "Type",
```
Comment (gopidesupavan) [inline airflow/ui/openapi-gen/requests/schemas.gen.ts]:
```suggestion
      title: Type,
```
doest it need to be ?

**Review by gopidesupavan — APPROVED** [review]

**Comment by bbovenzi** [comment]
These are autogenerated files. Is this a manual change or is it fixing an accidental manual change?

**Comment by gopidesupavan** [comment]
Hope this change is not required, it looks like the side effect of pydantic version issue?

This is fine now, https://github.com/apache/airflow/pull/44249

**Comment by amoghrajesh** [comment]
Yep it is due to the underlying pydantic bug fixed by https://github.com/apache/airflow/pull/44249. Closing this one

**Pull request closed without merging**